app/Models/CategoryProduct.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class CategoryProduct extends Model
+{
+    use HasFactory;
+
+    protected $fillable=['name','description','status'];
+
+    public function product(){
```

Comment:
harusnya nama functionnya itu products, karena ini relasi one (category) to many (products)

Reply:
sudah mas